src/debug/daccess/CMakeLists.txt

```diff
@@ -37,17 +46,19 @@ set(DACCESS_SOURCES
   datatargetadapter.cpp
 )
 
-if(IS_64BIT_BUILD EQUAL 1)
+if(CLR_CMAKE_PLATFORM_ARCH_AMD64)
   include_directories(amd64)
 
   list(APPEND DACCESS_SOURCES
     amd64/primitives.cpp
   )
-else()
+elseif(CLR_CMAKE_PLATFORM_ARCH_ARM)
+  include_directories(arm)
+
   list(APPEND DACCESS_SOURCES
-    i386/primitives.cpp
+    arm/primitives.cpp
```

Comment:
This seems to inadvertently remove the (basic blocks) of i386 build support.  Can we keep that intact please?